TrackingTools/DetLayers/interface/DetGroup.h

```diff
@@ -43,7 +43,10 @@ class DetGroup : public std::vector<DetGroupElement> {
   typedef std::vector<DetGroupElement> Base;
   typedef DetGroupElement::DetWithState DetWithState;
 
-  DetGroup() {}
+  DetGroup() {
+    index_ = 0;
+    indexSize_ = 0;
+  }
```

Comment:
```suggestion
  DetGroup() : index_(0), indexSize_(0) { }
```
initializer list is usually preferred, IIRC.